Sources/CvGame.cpp

```diff
@@ -118,6 +118,65 @@ CvGame::~CvGame()
 /************************************************************************************************/
 }
 
+namespace {
+	//TB GameOption compatibility enforcement project
+	void enforceOptionCompatability(CvGame* game)
+	{
+		for (int iI = 0; iI < NUM_GAMEOPTION_TYPES; iI++)
+		{
+			const GameOptionTypes eGameOption = static_cast<GameOptionTypes>(iI);
+			if (game->isOption(eGameOption))
+			{
+				CvGameOptionInfo& optionInfo = GC.getGameOptionInfo(eGameOption);
+
+				const int numOptionsForcedOn = optionInfo.getNumEnforcesGameOptionOnTypes();
+				if (numOptionsForcedOn > 0)
```

Comment:
This `if` is pointless (and the one below it), if the value is 0 the loop won't do anything anyway.